application/service/services.go

```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 
 	"github.com/fabric8-services/fabric8-cluster/cluster/repository"
+
+	uuid "github.com/satori/go.uuid"
```

Comment:
`uuid` is redundant

Reply:
yes, but it seems to be automatically added by the go tools. Is it a problem with your IDE?

Reply:
no, I am using Goland. I don't have this issue.